src/main/java/org/mule/service/http/impl/service/server/HttpServerManager.java

```diff
@@ -50,6 +50,24 @@ HttpServer createServerFor(ServerAddress serverAddress, Supplier<Scheduler> sche
                              Supplier<Long> shutdownTimeout)
       throws ServerCreationException;
 
+  /**
+   *
+   * @param serverAddress            address of the server
+   * @param schedulerSupplier        work manager source to use for retrieving an {@link Executor} for processing this server
+   *                                 requests
+   * @param usePersistentConnections if true, the connections will be kept open for subsequent requests
+   * @param connectionIdleTimeout    the amount of milliseconds to keep open an idle connection @return the create Server handler
+   * @param identifier               the id of the server
+   * @param shutdownTimeout          time to wait for persistent connections to be closed when server is stopped.
+   * @param readTimeout              time to wait while reading input in milliseconds
+   * @throws ServerCreationException if it was not possible to create the Server. Most likely because the host and port is already
+   *                                 in use.
+   */
+  HttpServer createServerFor(ServerAddress serverAddress, Supplier<Scheduler> schedulerSupplier,
```

Comment:
I think you need to add these methods in http-ee-service as well. Please check it

Reply:
Checked and the methods included on http-ee-service are `createServerManager` and `createManagedServer` which have not been modified on this PR